pkg/api/api.go

```diff
@@ -47,7 +50,33 @@ import (
 
 func dial(rpcServer string) (*grpc.ClientConn, error) {
 	// Set up and test connection to rpc server
-	creds := insecure.NewCredentials()
+	var creds credentials.TransportCredentials
+	tlsCACertFile := viper.GetString("tls_ca_cert")
+	useSystemTrustStore := viper.GetBool("trillian_log_server.tls")
+
+	switch {
+	case useSystemTrustStore:
+		creds = credentials.NewTLS(&tls.Config{
+			ServerName: rpcServer,
```

Comment:
it's unlikely, but possible that `rpcServer` will not simply be a `hostname/IP address` or `hostname:port` combination, as that string can be a value as defined by https://github.com/grpc/grpc/blob/master/doc/naming.md